ci-operator/config/openshift/file-integrity-operator/openshift-file-integrity-operator-master.yaml

```diff
@@ -70,6 +78,28 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-rosa
+  skip_if_only_changed: ^.*(md|adoc)$|^LICENSE$
+  steps:
+    cluster_profile: quay-aws
+    env:
+      BYO_OIDC: "true"
+      CHANNEL_GROUP: stable
+      OCM_LOGIN_ENV: integration
+      OPENSHIFT_VERSION: 4.15.8
+      REGION: us-east-2
+    test:
+    - as: test-rosa
+      cli: latest
+      commands: make e2e
```

Comment:
This may require an adjustment to modify the subscription so that the operator installs on only worker nodes.